addons/l10n_it_edi/tests/test_edi_import.py

```diff
@@ -71,6 +72,28 @@ def test_receive_signed_vendor_bill(self):
             }],
         }])
 
+    def test_receive_bill_sequence(self):
+        """ Ensure that the received bill gets assigned the right sequence. """
+        def mock_commit(self):
+            pass
+
+        filename = 'IT01234567890_FPR02.xml'
+        with (patch.object(self.proxy_user.__class__, '_decrypt_data', return_value=self.fake_test_content),
+              patch.object(sql_db.Cursor, "commit", mock_commit),
+              tools.mute_logger("odoo.addons.l10n_it_edi.models.account_move"),
```

Comment:
```suggestion
```

In the `test_receive_same_vendor_bill_twice` test we have a warning that says that we're ... importing twice. 

![image](https://github.com/odoo/odoo/assets/1665365/b5bcf839-ea19-4749-935b-e2ea904535f8)


I don't think it's needed here.